ocs_ci/utility/service.py

```diff
@@ -0,0 +1,245 @@
+import logging
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.ocs.ocp import wait_for_cluster_connectivity
+from ocs_ci.utility.utils import TimeoutSampler, exec_cmd
+
+logger = logging.getLogger(__name__)
+
+ACTIVE = b"active"
+INACTIVE = b"inactive"
+FAILED = b"failed"
+
+
+class Service(object):
+    """
+    Generic Service class
+
+    The purpose of this class is to provide a support to perform start/stop/restart and status
+    operations on a given service. The class is instantiated with required service name without
+    '.service extension. Refer to KubeletService class in this module for an example on how to
+    use these methods.
+    """
+
+    def __init__(self, service_name, force=False):
+        """
+        Class Initialization.
+
+        Initialize the service name local variable and collect a dictionary of Internel IP addresses
+        of nodes with node name as keys.
+        """
+        self.service_name = service_name
+        self.force = force
```

Comment:
I think you can get rid this option in this class,  The class doesn't need to be aware of this parameter.

Reply:
unless this `force` option is different from what is mentioned above.

Reply:
Vasu, see my comment above, the force is used as a boolean parameter.